### PR TITLE
Bsongis/ccm memory on f4

### PR DESCRIPTION
**Pull request by bsongis**
Ready for merge, it saves 45k RAM!

### CODE REVIEW

File: radio/src/opentx.cpp [hunk 1]
```diff
@@ -1453,7 +1453,7 @@ void flightReset()
 }
 
 #if defined(THRTRACE)
-uint8_t  s_traceBuf[MAXTRACE];
+uint8_t  s_traceBuf[MAXTRACE] _CCM;
```
Comment (bsongis) [inline radio/src/opentx.cpp]: Obviously this one is not reset to 0

File: radio/src/telemetry/telemetry.cpp [hunk 2]
```diff
@@ -1,6 +1,6 @@
 #include "../opentx.h"
 
-TelemetryItem telemetryItems[MAX_SENSORS];
+TelemetryItem telemetryItems[MAX_SENSORS] _CCM;
```
Comment (bsongis) [inline radio/src/telemetry/telemetry.cpp]: I am afraid the constructor is not called

**Comment by bsongis** [comment]
We need a way to initialize variables in CCM or reset them to 0 if not initialized, or call the constructors if any!

**Comment by bsongis** [comment]
Also I would like that all variables come to CCM except those marked as _NOCCM

**Comment by bsongis** [comment]
I will test this one as soon as I receive the X9E

**Comment by bsongis** [comment]
The radio is here. I can tell you, it is really nice!

**Comment by projectkk2glider** [comment]
> The radio is here. I can tell you, it is really nice!

Lucky you :grin:  ... now, for the punishment, you have to make sure that everything works.

**Comment by bsongis** [comment]
First flight done in my hattic :)

**Pull request merged**